images/ose-frr.yml

```
@@ -16,5 +16,6 @@ for_payload: false
 from:
   member: openshift-base-rhel8
 name: openshift/ose-frr
+name_in_bundle: metallb-frr-rhel8
```

hm. the actual image name needs to match what we're going to ship it as, which is [frr-rhel8](https://comet.engineering.redhat.com/containers/repositories/6177f5fa40a2d8e95c82d74b). I think we could just change the name.

`name_in_bundle` is to match the tag in the bundle image-references, which is really arbitrary. We would need it anyway since the name doesn't start with `ose-`.
```suggestion
name: openshift/frr-rhel8
name_in_bundle: metallb-frr-rhel8
```